picus/src/expr/impls/binary.rs

```diff
@@ -220,6 +220,8 @@ impl ConstraintExpr for BinaryExpr<Boolean> {
             match self.op() {
                 Boolean::And => lhs && rhs,
                 Boolean::Or => lhs || rhs,
+                Boolean::Implies => lhs && rhs,
+                Boolean::Iff => lhs && rhs,
```

Comment:
Is there a reason we cannot use `is_constant_false()` here? If we can use it, these could capture more cases by renaming the variables at lines 218-219 and defining these as follows:
```
Implies => lhs.is_constant_false() || (lhs.is_constant_true() && rhs.is_constant_true())
Iff => (lhs.is_constant_false() && rhs.is_constant_false()) || (lhs.is_constant_true() && rhs.is_constant_true())
```

Reply:
That's a good point. I didn't think of that

Reply:
Yeah I think this is better. Right now we only capture a subset of the cases where the boolean is true.